interface/grpc/core/list_services.go

```diff
@@ -2,8 +2,11 @@ package core
 
 import (
 	"context"
+	"sync"
 
 	"github.com/mesg-foundation/core/protobuf/coreapi"
+	service "github.com/mesg-foundation/core/service"
```

Comment:
One small change, remove service as it's already the name of pkg

Reply:
done